src/backend/distributed/sql/udfs/any_value/latest.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE FUNCTION pg_catalog.any_value_agg ( anyelement, anyelement )
+RETURNS anyelement AS $$
+        SELECT CASE WHEN $1 IS NULL THEN $2 ELSE $1 END;
+$$ LANGUAGE SQL STABLE;
+
+CREATE AGGREGATE pg_catalog.any_value (
+        sfunc    = pg_catalog.any_value_agg,
+        basetype = anyelement,
+        stype    = anyelement
```

Comment:
Can you add combinefunc = pg_catalog.any_value_agg? Then it can go through the new aggregate logic.